code/_helpers/text.dm

```diff
@@ -398,8 +398,8 @@ proc/TextPreview(var/string,var/len=40)
 	t = replacetext(t, "\[/i\]", "</I>")
 	t = replacetext(t, "\[u\]", "<U>")
 	t = replacetext(t, "\[/u\]", "</U>")
-	t = replacetext(t, "\[time\]", "[stationtime2text()]")
-	t = replacetext(t, "\[date\]", "[stationdate2text()]")
+	t = replacetext(t, "\[time\]", "[stationTimestamp()]")
+	t = replacetext(t, "\[date\]", "[stationTimestamp()]")
```

Comment:
these were two different values before, merge them if only one is being used now

what is the readout from stationtimestamp ?

Reply:
> these were two different values before, merge them if only one is being used now
> 
> 
> 
> what is the readout from stationtimestamp ?
> 
> 

Forget add "dd" and "hh"

Reply:
Fixed